docs/standard/commandline/get-started-tutorial.md

```diff
@@ -65,84 +65,94 @@ Create a .NET 6 console app project named "scl".
+* If no value was provided for `--file`, it prints available parse errors and returns `1` exit code:
 
-* Displays the contents of the specified file when the root command is invoked:
+:::code language="csharp" source="snippets/get-started-tutorial/csharp/Stage0/Program.cs" id="errors" :::
 
-  :::code language="csharp" source="snippets/get-started-tutorial/csharp/Stage1/Program.cs" id="handler" :::
+* The `ReadFile` method reads the specified file and displays its contents on the console:
+
+:::code language="csharp" source="snippets/get-started-tutorial/csharp/Stage0/Program.cs" id="action" :::
 
 ## Test the app
 
 You can use any of the following ways to test while developing a command-line app:
 
-* Run the `dotnet build` command, and then open a command prompt in the *scl/bin/Debug/net6.0* folder to run the executable:
+* Run the `dotnet build` command, and then open a command prompt in the *scl/bin/Debug/net9.0* folder to run the executable:
 
   ```console
   dotnet build
-  cd bin/Debug/net6.0
+  cd bin/Debug/net9.0
   scl --file scl.runtimeconfig.json
   ```
 
 * Use `dotnet run` and pass option values to the app instead of to the `run` command by including them after `--`, as in the following example:
 
   ```dotnetcli
-  dotnet run -- --file bin/Debug/net6.0/scl.runtimeconfig.json
-  ```
-
-The working directory is the project folder (the folder that has the .csproj  file), so the relative path to `scl.runtimeconfig.json` is from the project folder.
-
-  In .NET 7.0.100 SDK Preview, you can use the `commandLineArgs` of a *launchSettings.json* file by running the command `dotnet run --launch-profile <profilename>`.
-
-* [Publish the project to a folder](../../core/tutorials/publishing-with-visual-studio-code.md), open a command prompt to that folder, and run the executable:
-
-  ```console
-  dotnet publish -o publish
-  cd ./publish
-  scl --file scl.runtimeconfig.json
+  dotnet run -- --file bin/Debug/net9.0/scl.runtimeconfig.json
   ```
 
-* In Visual Studio 2022, select **Debug** > **Debug Properties** from the menu, and enter the options and arguments in the **Command line arguments** box. For example:
-
-  :::image type="content" source="media/get-started-tutorial/cmd-line-args.png" alt-text="Command line arguments in Visual Studio 2022":::
-
-  Then run the app, for example by pressing Ctrl+F5.
-
 This tutorial assumes you're using the first of these options.
 
 When you run the app, it displays the contents of the file specified by the `--file` option.
 
 ```output
 {
   "runtimeOptions": {
-    "tfm": "net6.0",
+    "tfm": "net9.0",
     "framework": {
       "name": "Microsoft.NETCore.App",
-      "version": "6.0.0"
+      "version": "9.0.0"
     }
   }
 }
 ```
 
-### Help output
+But what happens if you ask it to display the help by providing `--help`? Nothing gets printed to the console, because the app doesn't yet handle a scenario, where `--file` is not provided and there are no parse errors.
```

Comment:
```suggestion
But what happens if you ask it to display the help by providing `--help`? Nothing gets printed to the console, because the app doesn't yet handle the scenario where `--file` is not provided and there are no parse errors.
```